modin/pandas/test/dataframe/test_binary.py

```diff
@@ -90,6 +90,15 @@ def test_math_functions(other, axis, op):
     )
 
 
+@pytest.mark.parametrize("other", [lambda df: 2, lambda df: df])
+def test___divmod__(other):
+    data = test_data["float_nan_data"]
+
+    eval_general(*create_test_dfs(data), lambda df: divmod(df, other(df)))
+    # test __rdivmod__
+    eval_general(*create_test_dfs(data), lambda df: divmod(other(df), df))
```

Comment:
Why should we test two different methods and in a single test?

Reply:
Splitted